hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -431,6 +431,10 @@ public StorageLevel getBloomIndexInputStorageLevel() {
     return StorageLevel.fromString(props.getProperty(HoodieIndexConfig.BLOOM_INDEX_INPUT_STORAGE_LEVEL));
   }
 
+  public boolean getBloomIndexShouldUpdatePartitionPath() {
```

Comment:
would appreciate if you can add java docs

Reply:
@nsivabalan I think for a simple a getter that fetch the property, the javadoc is accessible from the property itself. I've added the docs there. So can i skip it here?